warehouse/query-core/src/main/java/datawave/query/tld/TLDQueryIterator.java

```diff
@@ -268,4 +273,10 @@ protected IteratorBuildingVisitor createIteratorBuildingVisitor(final Range docu
                         TLDIndexIteratorBuilder.class);
     }
     
+    @Override
+    protected Function<Tuple2<Key,Document>,Tuple3<Key,Document,Map<String,Object>>> buildTfFunction(TermFrequencyConfig tfConfig) {
```

Comment:
Is there a way to incorporate the changes into the TLDQueryIterator unit tests to check how everything hooks up in the new configuration?

Reply:
Added in https://github.com/NationalSecurityAgency/datawave/pull/1773/commits/a516807fc75d1db36540540c57ccaf65788a901b